src/HornedBeast.js

```diff
@@ -1,10 +1,39 @@
 import React from 'react';
+import Card from 'react-bootstrap/Card';
+
+import './HornedBeast.css';
 
 class HornedBeast extends React.Component {
-  render () {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      votes: 0,
+    }
+  }
+
+  handleVote = () => {
+    this.setState({
+      votes: this.state.votes + 1
+    })
+  }
+
+
+  render() {
     return (
-      <>
-      </>
+      <article>
+        <Card style={{ width: '24rem' }}>
+          <Card.Img variant="top"
+            src={this.props.imageUrl}
+            alt={this.props.description}
+            title={this.props.title}
+            onClick={this.handleVote} />
+          <Card.Body className="p-3 mb-2 bg-info text-white">
+            <h3>{this.props.title}</h3>
+            Votes :{this.state.votes}
```

Comment:
Awesome job on getting votes to increment! All that is left here is:

- Put a heart in each horned beast with the number of times it was “favorited” next to it. 👍